sp/src/game/client/workshop_publish.cpp

```diff
@@ -341,6 +342,12 @@ class CPreviewImage : public Panel
 				text = g_pVGuiLocalize->Find( "#WorkshopMgr_NoImage" );
 			}
 
+			if (text == NULL)
+			{
+				Msg( "ERROR: Could not localize text string for preview image!\n" );
+				return;
```

Comment:
This is unnecessary and unideal (inside `Paint()`). It would be better to fallback to either the original string `text = L"#WorkshopMgr_NoImage"` as is the case for how vgui panels handle these, or the English string `text = L"no image"`